person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initilize(age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] You made a mistake here. I think you meant `initialize` here. Kindly fix this.